src/EditorFeatures/CSharpTest/CodeActions/ConvertLocalFunctionToMethod/ConvertLocalFunctionToMethodTests.cs

```diff
@@ -498,7 +499,7 @@ private static async void LocalFunction<T>(CancellationToken c, Task<T> task)
 }");
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertLocalFunctionToMethod)]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertLocalFunctionToMethod), WorkItem(35180, "https://github.com/dotnet/roslyn/issues/35180")]
```

Comment:
i don't understand why the existing tests were updated with workitems.

Reply:
The existing tests were updated to reflect changes done for the Workitem. Should I remove it?

Reply:
if there was no change in their bheavior, i would revert.

Reply:
There was change in behavior of the Tests. Some that tested no-refactorings were even updated to test presence of one.

Reply:
but i don't see any edits to these tests... so the behavior looks like it hasn't changed.

Reply:
They changed in previous PR.

Reply:
ahhhh.  In that case, this is fine.  Sorry, that wasn't at all clear here :)